modules/op-results-cert.adoc

```diff
@@ -0,0 +1,40 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="results-cert_{context}"]
+= Preparing a secret with an SSL certificate
+
+{tekton-results} provides a REST API using the HTTPS protocol, which requires an SSL certificate. Provide a secret with this certificate. If you have an existing certificate provided by a certificate authority (CA), use this certificate, otherwise create a self-signed certificate.
+
+.Prerequisites
+
+* The `openssl` command-line utility is installed.
+
+.Procedure
+
+. If you do not have a certificate provided by a CA, create a self-signed certificate by running the following command:
++
+[source,bash]
+----
+$ openssl req -x509 \
```

Comment:
Remove "$", so that user can easily copy.

Reply:
Impossible, unfortunately. We have a style requirement to have the $ in terminal commands.